plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -66,12 +226,96 @@ func (c *CiscoTelemetryMDT) Start(acc telegraf.Accumulator) error {
+{"Name": "sys/bgp","prop": [{"Key": "dynRtMap","Value": "string"}, {"Key": "nhRtMap","Value": "string"}, {"Key": "epePeerSet","Value": "string"}, {"Key": "asn","Value": "string"}, {"Key": "peerImp","Value": "string"}, {"Key": "wght","Value": "string"}, {"Key": "assocDom","Value": "string"}, {"Key": "tblMap","Value": "string"}, {"Key": "unSupprMap","Value": "string"}, {"Key": "sessionContImp","Value": "string"}, {"Key": "allocLblRtMap","Value": "string"}, {"Key": "defMetric","Value": "string"}, {"Key": "password","Value": "string"}, {"Key": "retainRttRtMap","Value": "string"}, {"Key": "clusterId","Value": "string"}, {"Key": "localAsn","Value": "string"}, {"Key": "name","Value": "string"}, {"Key": "defOrgRtMap","Value": "string"}]},
+{"Name": "sys/ch","prop": [{"Key": "fanName","Value": "string"}, {"Key": "typeCordConnected","Value": "string"}, {"Key": "vendor","Value": "string"}, {"Key": "model","Value": "string"}, {"Key": "rev","Value": "string"}, {"Key": "vdrId","Value": "string"}, {"Key": "hardwareAlarm","Value": "string"}, {"Key": "unit","Value": "string"}, {"Key": "hwVer","Value": "string"}]},
+{"Name": "sys/intf","prop": [{"Key": "descr","Value": "string"}, {"Key": "name","Value": "string"}, {"Key": "lastStCause","Value": "string"}, {"Key": "description","Value": "string"}, {"Key": "unit","Value": "string"}, {"Key": "operFECMode","Value": "string"}, {"Key": "operBitset","Value": "string"}, {"Key": "mdix","Value": "string"}]},
+{"Name": "sys/procsys","prop": [{"Key": "name","Value": "string"}, {"Key": "id","Value": "string"}, {"Key": "upTs","Value": "string"}, {"Key": "interval","Value": "string"}, {"Key": "memstatus","Value": "string"}]},
+{"Name": "sys/proc","prop": [{"Key": "processName","Value": "string"}, {"Key": "procArg","Value": "string"}]},
+{"Name": "sys/bfd/inst","prop": [{"Key": "descr","Value": "string"}, {"Key": "vrfName","Value": "string"}, {"Key": "name","Value": "string"}, {"Key": "name","Value": "string"}]},
+{"Name": "sys/lldp","prop": [{"Key": "sysDesc","Value": "string"}, {"Key": "portDesc","Value": "string"}, {"Key": "portIdV","Value": "string"}, {"Key": "chassisIdV","Value": "string"}, {"Key": "sysName","Value": "string"}, {"Key": "name","Value": "string"}, {"Key": "id","Value": "string"}]}]`)
```

Comment:
Can we please move this to its own file!? This clutters the file and makes reading difficult.